examples/mysql-backup-create-service-account/main.tf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "mysql" {
+  source               = "../../modules/mysql"
+  name                 = var.db_name
+  database_version     = "MYSQL_8_0"
+  random_instance_name = true
+  project_id           = var.project_id
+  zone                 = "${var.region}-a"
+  region               = var.region
+  deletion_protection  = false
+
+  ip_configuration = {
+    ipv4_enabled        = true
+    private_network     = null
+    require_ssl         = true
+    allocated_ip_range  = null
+    authorized_networks = var.authorized_networks
+  }
+}
+
+resource "google_storage_bucket" "backup" {
+  name     = "${module.mysql.instance_name}-backup"
+  location = var.region
+  # TODO: don't use force_destroy for production this is just required for testing
+  force_destroy = true
+  project       = var.project_id
+}
+
+module "backup" {
+  source                = "../../modules/backup"
+  region                = var.region
```

Comment:
nit: we are trying to reduce from variables in examples. It would be ideal if we just need project_id as a var